Makefile

```diff
@@ -182,10 +182,6 @@ vendor: $(GVT)
 	@echo restoring vendor
 	$(GVT) restore
 
-integration-tests: $(GOVC) components isos vic-machine
```

Comment:
@emlin I don't think this is feasible at this point.  The integration tests require much more orchestrating now, with the custom image, environment variables and drone secrets.  I think at this point the only mechanism we should support is the drone methodology.